docs/Walkthrough.md

```diff
@@ -1,88 +1,181 @@
 # BTCPay Server Walkthrough
 
-In this article, we will walk you through the **BTCPay user interface** and show you how to navigate the different options.
+This article will walk you through the **BTCPay user interface** and show you how to navigate the different options.
 
-Watch video below for an interactive overview of the features.
+Watch the video below for an interactive overview of the features.
 
 [![BTCPay Server Walkthrough](https://img.youtube.com/vi/ZIfJyq9RimM/mqdefault.jpg)](https://www.youtube.com/watch?v=ZIfJyq9RimM "BTCPay Server - WalkThrough")
```

Comment:
Considering that this video is now completely outdated, it's probably a good idea to remove it entirely until we have a new one.
```suggestion
Watch the video below for an interactive overview of the features.

[![BTCPay Server Walkthrough](https://img.youtube.com/vi/ZIfJyq9RimM/mqdefault.jpg)](https://www.youtube.com/watch?v=ZIfJyq9RimM "BTCPay Server - WalkThrough")
```
```suggestion
Watch the video below for an interactive overview of the features.

[![BTCPay Server Walkthrough](https://img.youtube.com/vi/ZIfJyq9RimM/mqdefault.jpg)](https://www.youtube.com/watch?v=ZIfJyq9RimM "BTCPay Server - WalkThrough")
```